package.json

```diff
@@ -8,6 +8,10 @@
     "node": ">=22.0.0 <23.0.0",
     "npm": ">=10.9.0 <12.0.0"
   },
+  "files": [
```

Comment:
this should not be needed. instead move the `static` folder to the project root and include the files for packaging to the lambda function in the `wsk` part like so:
```json
    "static": [
      "static/profiles.json"
    ]
```